addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -28,13 +28,13 @@
                 </t>
                 <div class="o-autogrow"/>
                 <div class="o_ChatWindowHeader_item o_ChatWindowHeader_rightArea">
-                    <t t-if="chatWindow.hasShiftLeft">
-                        <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandShiftLeft" t-att-class="{ 'o-mobile': env.messaging.device.isMobile }" t-on-click="_onClickShiftLeft" title="Shift left">
+                    <t t-if="chatWindow.hasShiftPrev">
+                        <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandShiftPrev" t-att-class="{ 'o-mobile': env.messaging.device.isMobile }" t-on-click.stop="_onClickShiftPrev" t-att-title="shiftPrevText">
                             <i class="fa fa-angle-left"/>
```

Comment:
It this one of those magic where we inverse the direction of fa icon depending on text direction?

Because otherwise this would look wrong. "shift prev" should be "left arrow" in LTR but it should be "right arrow" in RTL (similar to how we change the text)

Reply:
Yes it's "magic", I changed it and the arrows were not in the right direction .. so we should just leave them like that

Reply:
Ok, but then I would add a short comment in the code to explain this.